src/snowflake/cli/_plugins/spcs/compute_pool/commands.py

```diff
@@ -61,6 +55,8 @@ def _compute_pool_name_callback(name: FQN) -> FQN:
     """
     Verifies that compute pool name is a single valid identifier.
     """
+    if not name:
+        return name
     if not is_valid_object_name(name.identifier, max_depth=0, allow_quoted=False):
```

Comment:
```suggestion
if name and not is_valid_object_name(name.identifier, max_depth=0, allow_quoted=False):
```
This would make the first check not needed, if i'm right

Reply:
You are right 😄